### PR TITLE
FOPTS-5507  Added hide_skip_approvals to All Active Policies

### PR DESCRIPTION
**Pull request by ramyadmz**
### Description
This PR marks the last update for the Hide Skip Approval [task](https://flexera.atlassian.net/browse/FLEX-5397).
✅ The first phase, updating AWS Right Size policies, is complete in [PR-2799](https://github.com/flexera-public/policy_templates/pull/2799).
✅ Additionally, the metaparent policy compiler and Rakefile have been successfully updated : [PR-2813](https://github.com/flexera-public/policy_templates/pull/2813)

Current Phase (Phase 2):
In this phase, we have added the hide_skip_approvals field to the info section of policy templates that do not use the request approval option in any escalation. This field allows the UI to dynamically show or hide the "Skip Approval" option based on the policy’s configuration. Following the AWS Right Size policy updates, these changes have now been applied to **all remaining active policies.**

More context:[ Seeking Your Input: Proposed Solution for Conditional "Skip Approvals" Visibility](https://teams.microsoft.com/l/message/19:833373548e104af2a20b0216eda1ba7b@thread.skype/1728495063426?tenantId=91034d23-0b63-4943-b138-367d4dfac252&groupId=fb250818-e040-4a26-b207-61c3cd99fd6e&parentMessageId=1728495063426&teamName=Team%20Flexera%20One&channelName=Policy%20Support%20and%20Questions&createdTime=1728495063426)

### Issues Resolved

[FLEX-5397](https://flexera.atlassian.net/browse/FLEX-5397)

### Contribution Check List

- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: security/aws/aws_config_enabled/aws_config_enabled.pt [hunk 1]
```diff
@@ -16,6 +16,8 @@ info(
   benchmark_control: "3.5",
   benchmark_version: "1.4.0",
   cis_controls: "[\"1.1v8\", \"1.4v7\", \"11.2v7\", \"16.1v7\"]",
+  hide_skip_approvals: "true",
+
```
Comment (XOmniverse) [inline security/aws/aws_config_enabled/aws_config_enabled.pt:20]: Extraneous empty line should be removed

**Review by XOmniverse — CHANGES_REQUESTED** [review]

File: security/aws/aws_config_enabled/aws_config_enabled.pt (hunk 1 above)
Comment (ramyadmz) [reply security/aws/aws_config_enabled/aws_config_enabled.pt]: applied at https://github.com/flexera-public/policy_templates/pull/2830/commits/36db4746fcd3b4771d4e66462013e6892ff82fc5

File: cost/turbonomics/scale_virtual_machines_recommendations/aws/turbonomics_scale_virtual_machines.pt [hunk 2]
```diff
@@ -134,7 +135,7 @@ end
 ##this will potentially be a lot of calls. how to make this more performant
 datasource "ds_get_business_units" do
   request do
-    run_script $js_get_business_units, $param_turbonomic_endpoint, $param_provider, val($ds_get_turbonomic_token, "access_token")
+    run_script $js_get_business_units,val($ds_get_turbonomic_token, "access_token"), $param_turbonomic_endpoint, $param_provider
```
[the remark this answers is not in the document]
Comment (ramyadmz) [reply cost/turbonomics/scale_virtual_machines_recommendations/aws/turbonomics_scale_virtual_machines.pt]: applied at https://github.com/flexera-public/policy_templates/pull/2830/commits/36db4746fcd3b4771d4e66462013e6892ff82fc5

File: cost/turbonomics/rightsize_virtual_volumes_recommendations/azure/turbonomics_rightsize_virtual_volumes_recommendations.pt [hunk 3]
```diff
@@ -124,7 +125,7 @@ end
 
 datasource "ds_get_business_units" do
   request do
-    run_script $js_get_business_units, $param_turbonomic_endpoint, $param_provider, val($ds_get_turbonomic_token, "access_token")
+    run_script $js_get_business_units,val($ds_get_turbonomic_token, "access_token"), $param_turbonomic_endpoint, $param_provider
```
[the remark this answers is not in the document]
Comment (ramyadmz) [reply cost/turbonomics/rightsize_virtual_volumes_recommendations/azure/turbonomics_rightsize_virtual_volumes_recommendations.pt]: applied at https://github.com/flexera-public/policy_templates/pull/2830/commits/36db4746fcd3b4771d4e66462013e6892ff82fc5

File: cost/turbonomics/rightsize_virtual_volumes_recommendations/azure/turbonomics_rightsize_virtual_volumes_recommendations.pt [hunk 4]
```diff
@@ -95,7 +96,7 @@ end
 
 datasource "ds_get_turbonomic_recommendations" do
   request do
-    run_script $js_get_turbonomic_recommendations, $param_turbonomic_endpoint, val($ds_get_turbonomic_token, "access_token")
+    run_script $js_get_turbonomic_recommendations,val($ds_get_turbonomic_token, "access_token"), $param_turbonomic_endpoint
```
[the remark this answers is not in the document]
Comment (ramyadmz) [reply cost/turbonomics/rightsize_virtual_volumes_recommendations/azure/turbonomics_rightsize_virtual_volumes_recommendations.pt]: applied at https://github.com/flexera-public/policy_templates/pull/2830/commits/36db4746fcd3b4771d4e66462013e6892ff82fc5

File: cost/turbonomics/rightsize_databases_recommendations/azure/turbonomics_rightsize_databases_recommendations.pt [hunk 5]
```diff
@@ -336,7 +336,7 @@ end
 
 datasource "ds_get_pricing_model" do
   request do
-    run_script $js_get_pricing_model, $ds_only_uuids, $param_turbonomic_endpoint, val($ds_get_turbonomic_token, "access_token")
+    run_script $js_get_pricing_model,val($ds_get_turbonomic_token, "access_token"), $ds_only_uuids, $param_turbonomic_endpoint
```
[the remark this answers is not in the document]
Comment (ramyadmz) [reply cost/turbonomics/rightsize_databases_recommendations/azure/turbonomics_rightsize_databases_recommendations.pt]: applied at https://github.com/flexera-public/policy_templates/pull/2830/commits/36db4746fcd3b4771d4e66462013e6892ff82fc5

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**